content/posts/newsletter/2020-04-02.md

```diff
@@ -0,0 +1,143 @@
+
+Alexander Gnauck published the [Membership Applications Q2 2020](https://wiki.xmpp.org/web/Membership_Applications_Q2_2020). Applications are welcome!
+
+
+## Articles
+
+[Pasis](https://github.com/pasis), maintainer of libstrophe and contributor to profanity, wrote about using the [XML console in profanity](https://profanity-im.github.io/blog/post/xml-console-in-profanity/) for debugging.
+
+Jan Cieśla from MongooseIM wrote a blog post how [they thrive their project by the metrics](https://www.erlang-solutions.com/blog/how-data-drives-mongooseim.html).
+
+Can you believe? Zoom uses XMPP for its chat - cool! [Vulnerability](https://blog.talosintelligence.com/2020/04/zoom-user-enumeration.html) - not cool.
+
+## Software releases
+
+Dino has now [Last Message Correction](https://github.com/dino/dino/commit/871ff33ac79f3d17b0260b8bfcd27780038edd6d) for all the typos and regrets you made. And also [voice handleing in moderated groups](https://github.com/dino/dino/commit/2631a9bdbaf9a40f329f05c55c6e2ea38efeb10c)
+
+[Kaidan 0.5.0](https://www.kaidan.im/2020/04/06/kaidan-0.5.0/) - Bam! Check out their new onboarding!
+
+Lets say it like this: Removing OTR + Rework conference and contact details + show PDF previews + title for audio files = [Pix-Art Messenger 2.3.5](https://github.com/kriztan/Pix-Art-Messenger/blob/master/CHANGELOG.md)
+
+Four letters for Windows Users: [UWPX 0.25.0](https://github.com/UWPX/UWPX-Client/releases/tag/v.0.25.0.0) is online with Windows10X support (what ever that Windows 10 is) and a push server. So give it a try!
+
+Monal continues on fighting the iOS front: From the [beta](https://monal.im/blog/4-6-betas/) upcoming are support for geo location messages, [TLS ALPN support](https://github.com/anurodhp/Monal/issues/206) and improvements on [OpenFire push notifications](https://github.com/anurodhp/Monal/issues/354) as well as many fixes with smack.
+
+Wow - no late April joke - Conversations released version 2.8.0 with [audio and video calls](https://twitter.com/iNPUTmice/status/1254662039592742913)! Insane!
+
+
+### Servers
+
+Guus der Kinderen, from the Ignite Realtime Foundation Board, wrote a short article on [how to setup a STUN and TURN service in Openfire for audio and video calls](https://discourse.igniterealtime.org/t/preparing-openfire-for-audio-video-calls-with-conversations/87828).
+
+
+### Clients and applications
+
+The maintainer of libstrophe, [Pasis](https://github.com/pasis), and contributor to Profanity, created a tool called [xmppconsole](https://github.com/pasis/xmppconsole) which sends raw XMPP stanzas over an XMPP connection and displays the XMPP stream. Main purpose is to study XEPs and debug servers implementation.
+The tool is under development, the finished version will support both GTK UI and ncurses UI. In the blog they also explained how to easier contribute commits to their repository, [read on](https://profanity-im.github.io/blog/post/contributing-a-patch-via-github/)!
```

Comment:
development, the finished => development. The final
In the blog they => In their blog, they